src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -140,20 +145,28 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String messageContent = request.getParameter("message");
 
-    // this removes any HTML from the message content
-    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
-
+    // allows users to enter basic HTML tags that are not a threat to security
+    String HTMLMessageContent = clean(messageContent, Whitelist.basic());
+    
     Message message =
         new Message(
             UUID.randomUUID(),
             conversation.getId(),
             user.getId(),
-            cleanedMessageContent,
+            HTMLMessageContent,
             Instant.now());
 
     messageStore.addMessage(message);
 
     // redirect to a GET request
     response.sendRedirect("/chat/" + conversationTitle);
   }
+
+public static String clean(String cleanedMessage, Whitelist whitelist ) {
```

Comment:
Nit: If you want the whole codebase to match, you could indent this method definition by two spaces, then move the closing brace on line 171 out by two so it lines up. That's just a tiny aesthetic thing (and I'll check in the meeting how much you all want me to comment on these kinds of nits), but making everything pretty and matching helps keep large code bases tidy.

Your editor might also have some automatic code formatting settings--in Eclipse, it's Ctrl-Shift-F